executor/modes/local/websocket_client.py

```
@@ -0,0 +1,240 @@
+            wait_timeout: Maximum time to wait for connection in seconds.
+
+        Returns:
+            True if connected successfully, False otherwise.
+
+        Raises:
+            ValueError: If backend_url or auth_token is not configured.
+        """
+        if not self.backend_url:
+            raise ValueError(
+                "Backend URL not configured. Set WEGENT_BACKEND_URL environment variable."
+            )
```

Ruff (0.14.14)

Avoid specifying long messages outside the exception class

(TRY003)